example/ck_tile/38_block_scale_gemm/gemm_aquant_quantgrouped.cpp

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2025, Advanced Micro Devices, Inc. All rights reserved.
+
+// This example demonstrates 2D block scale quantization (N×K) for BQuant
```

Comment:
```suggestion
// This example demonstrates 2D block scale quantization (N×K) for BQuant
```
This comment is no longer relevant in this file now.